src/Service/ProgrammesAggregationService.php

```diff
@@ -5,6 +5,7 @@
 use BBC\ProgrammesCachingLibrary\CacheInterface;
 use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ClipRepository;
 use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+use BBC\ProgrammesPagesService\Domain\ApplicationTime;
```

Comment:
Is this used?

Reply:
No but I ran `git revert` on the commits that added this functionality, which included this bit of tidy up. Have added an extra commit to remove this again.